cartographer_ros/cartographer_ros/node.cc

```diff
@@ -176,36 +176,36 @@ void Node::AddSensorSamplers(const int trajectory_id,
           options.landmarks_sampling_ratio));
 }
 
-void Node::PublishTrajectoryStates(const ::ros::TimerEvent& timer_event) {
+void Node::PublishLocalTrajectoryData(const ::ros::TimerEvent& timer_event) {
   carto::common::MutexLocker lock(&mutex_);
-  for (const auto& entry : map_builder_bridge_.GetTrajectoryStates()) {
-    const auto& trajectory_state = entry.second;
+  for (const auto& entry : map_builder_bridge_.GetLocalTrajectoryData()) {
+    const auto& trajectory_data = entry.second;
```

Comment:
I think it's fine to use trajectory_data here for brevity, it's clear from the context that it's local.